src/webui/yarn.lock

```diff
@@ -1898,6 +1898,13 @@ copy-descriptor@^0.1.0:
   version "0.1.1"
   resolved "https://registry.yarnpkg.com/copy-descriptor/-/copy-descriptor-0.1.1.tgz#676f6eb3c39997c2ee1ac3a924fd6124748f578d"
 

```

Comment:
Please add at least one config-test for a new tuner.

Reply:
config-test added